### DISCUSSION TITLE
How to use the datejs library?

### DISCUSSION THREAD

**Discussion by StephanBergmann** [neuefische/questions#748] [Web] [answered]
Hello together,

we want to use the datejs library from npm (already installed) but don't know how to import it in a component, so that we can use it. We have searched the documentation from datejs and the internet  but didn't find something useful.

Maybe you know it or have an idea.

Thank you in advance.

**Comment by moonwave99** [comment, marked as answer]
We recommend [date-fns](https://www.npmjs.com/package/date-fns) over Date.js as it is mode modern / maintained.

What is the concrete use case though? A lot can be achieved with [Date.prototype.toLocaleDateString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString) nowadays.

[A collection of useful use cases can be found here as well](https://codesandbox.io/s/dealing-with-dates-8vlgth) - hope it helps!

**Comment by onemanwenttomow** [comment]
I just had a discussion with @StephanBergmann and the rest of the team where we got to the root of the problem. (The seeded dates are in a different format to the what is now being created in the app.)

I think the `datejs` is a little outdated. 

Like @moonwave99 I would also suggest https://date-fns.org/

Another alternative would be https://day.js.org/
